### ISSUE TITLE
Add support for labels

### ISSUE THREAD

**Issue by albertschwarzkopf** [AndreasAugustin/actions-template-sync#129]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
An option to add a label on the pull request.

**Comment by AndreasAugustin** [comment]
@albertschwarzkopf thanks a lot for the feature request. I think that would be a nice nehancement

**Comment by AndreasAugustin** [comment]
@all-contributors please add @albertschwarzkopf  for ideas

**Comment by AndreasAugustin** [comment]
The github cli is used to create a PR. According to [docs](https://cli.github.com/manual/gh_pr_create) it is possible to add labels to the `create` command.
Think it makes sense to extend the enhancement to be able to add one ore more labels to the PR.

@albertschwarzkopf  would be something like the following sufficient for you?

```yaml
# File: .github/workflows/template-sync.yml

on:
    # cronjob trigger
  schedule:
  - cron:  "0 0 1 * *"
  # manual trigger
  workflow_dispatch:
jobs:
  repo-sync:
    runs-on: ubuntu-latest

    steps:
      # To use this repository's private action, you must check out the repository
      - name: Checkout
        uses: actions/checkout@v2
      - name: actions-template-sync
        uses: AndreasAugustin/actions-template-sync@v0.1.6-draft
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          source_repo_path: <owner/repo>
          upstream_branch: <target_branch> # defaults to main
          pull_request_labels:  <label_1> [<label_2> ...] # defaults to template_sync chore
```

**Comment by albertschwarzkopf** [comment]
@AndreasAugustin  Thank you! Yes that would be perfect! In my case a single label is enough, but a list of labels would be better.

**Comment by AndreasAugustin** [comment]
@albertschwarzkopf thanks a lot for this suggestion.
There is a new draft version available [v0.2.0-draft](https://github.com/marketplace/actions/actions-template-sync)

Sorry that it took some time, am still super busy atm. (hope to find some more next year, have some nice ideas for this project).

Please check the [README.md](README.md) for the usage information.
